projects/storefrontlib/src/lib/cms-lib/navigation/navigation-ui.component.scss

```diff
@@ -1,189 +1,148 @@
 @import 'theme';
-@import 'functions';
-@import 'mixins';
-@import '~bootstrap/scss/nav';
-@import '~bootstrap/scss/dropdown';
```

Comment:
There is an unwanted side effect to removing this dropdown import (I think it's the cause) basically you will see on this branch the dropdown has a larger top margin. This import overrides a default 11px from a class `dropdown-menu` in blocks/dropdown. The margin is then 0.125em which imho looks better. You could manually add the top margin to `cx-nav-child-list-columns`. We can talk about it if you want me to show you.

Reply:
fixed! (without the import)